### ISSUE TITLE
Create an account on the mangoblogger.com website when user signs in the app

### ISSUE THREAD

**Issue by mangoblogger** [mangoblogger/MangoBloggerAndroidApp#5]

**Comment by andy1729** [comment]
Use existing website login in webview in app.

**Comment by mangoblogger** [comment]
can we verify the user once they login using webview and let them but things on mobile?

**Comment by andy1729** [comment]
Yes sir, based on the response user gets after submitting login for we can implement the logic to let them enter inside app. Using javascript, we can customise everything, including the design of login/register form  (just like I did when we were using webview for terms), the user will hardly notice anything. 

Let me create a demo for you.
